e2e-polybft/framework/test-cluster.go

```diff
@@ -51,6 +51,22 @@ const (
 	nonValidatorPrefix = "test-non-validator-"
 )
 
+type NodeType int
+
+const (
+	None      = 0
+	Validator = 1 << iota
```

Comment:
Is it better this way maybe?

```suggestion
	None      NodeType = 0
	Validator NodeType = 1 << iota
```